specification/liftrpinecone/Pinecone.VectorDb.Management/main.tsp

```diff
@@ -0,0 +1,90 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "@azure-tools/typespec-liftr-base";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.ResourceManager;
+using LiftrBase;
+
+/* Pinecone Resource Provider management API. */
+@armProviderNamespace
+@service({
+  title: "Pinecone management service",
+})
+@versioned(Pinecone.VectorDb.Versions)
+namespace Pinecone.VectorDb;
+
+/** The available API versions. */
+enum Versions {
+  /** 2024-10-22-preview version */
+  @useDependency(Azure.ResourceManager.Versions.v1_0_Preview_1)
+  @useDependency(Azure.Core.Versions.v1_0_Preview_2)
+  @armCommonTypesVersion(Azure.ResourceManager.CommonTypes.Versions.v5)
+  @useDependency(LiftrBase.Versions.v1_preview)
+  v2024_10_22_preview: "2024-10-22-preview",
+}
+
+@doc("Partner's specific Properties")
+model PartnerProperties {
+  @doc("Pinecone Organization Name")
+  @minLength(1)
+  @maxLength(512)
+  @pattern("^[^\\0]+$")
+  displayName: string;
+}
+
+/** These properties will be sent to your endpoints registered with Liftr during CRUD on resource.
+ * BaseResourceProperties contains Marketplace, useDetail related properties*/
+@doc("Properties specific to Organization")
+model OrganizationProperties is BaseResourceProperties {
+  @doc("partner properties")
+  partnerProperties?: PartnerProperties;
+
+  @doc("Single sign-on properties")
+  singleSignOnProperties?: SingleSignOnPropertiesV2;
+}
+
+model OrganizationResource
+  is Azure.ResourceManager.TrackedResource<OrganizationProperties> {
+  @key("organizationname")
```

Comment:
>organizationname [](http://example.com/codeflow?start=8&length=16)

nit casing: organizationName

Reply:
if its not mandatory to change, can we have it like this. Issue is external partner has written this typespec and their endpoints are also defined the same way. If i will change the name, we might need to follow up with partner and retest their endpoints if they are not case insensitive.